spyder/preferences/shortcuts.py

```diff
@@ -808,12 +817,22 @@ def setup_page(self):
         self.table = ShortcutsTable(self, text_color=ima.MAIN_FG_COLOR)
         self.finder = ShortcutFinder(self.table, self.table.set_regex)
         self.table.finder = self.finder
+        self.table.finder.setPlaceholderText(
+            _("Search a shortcut in the table"))
+        self.table.setMouseTracking(True)
         self.label_finder = QLabel(_('Search: '))
         self.reset_btn = QPushButton(_("Reset to default values"))
+        self.label = QLabel()
+        self.label.setText(
+            _("Here you can browse the list of all available shortcuts in "
+              "Spyder. You can also customize them by double-clicking on any "
+              "entry in this table."))

```

Comment:
There's no need to create the label first and then set its text. You can do it in a single step, like this:

```python
self.top_label = QLabel(
    _("Here you can browse the list of all available shortcuts in "
    ...
)
```

And yes, please rename this variable to `top_label`, instead of simply `label`.